core/modules/user/user.module

```diff
@@ -1628,6 +1634,14 @@ function user_menu_alter(&$items) {
     // Set "Manage fields" tab as the default tab.
     $items['admin/config/people/manage/fields']['type'] = MENU_DEFAULT_LOCAL_TASK;
   }
+
+  // If we are using the simplified look for login pages, suppress the layout by
+  // providing a delivery_callback value. See menu_execute_active_handler().
+  if (config_get('system.core', 'user_login_appearance_setting') == 'simplified') {
```

Comment:
I know @indigoxela was annoyed by the existence of a utility function for this, but I did think it was neater to do `user_login_appearance_setting()` rather than calling `config_get()` repeatedly. Not a performance issue, just neater, hence, not an issue.

Reply:
I feel this way too. When we're checking this value in eight places in this PR, and other contrib modules might want to as well; wrapping it in function gives a place for documentation and can avoid typo problems.